src/test/java/io/appium/java_client/android/FingerPrintTest.java

```diff
@@ -0,0 +1,120 @@
+        }
+    }
+
+    /**
+     * finishing.
+     */
+    @AfterClass public static void afterClass() {
+        if (service != null) {
+            service.stop();
+        }
+    }
+
+    /**
+     * enable system security which is required for finger print activation.
+     */
+    @Before public void before() throws Exception {
+        final AndroidDriver driver = getAndroidDriver("ChooseLockGeneric");
+        TimeUnit.SECONDS.sleep(2);
+        // clicking the pin lock mode
+        driver.findElement(By.xpath("//android.widget.ListView[1]/android.widget.LinearLayout[4]"))
+            .click();
+        TimeUnit.SECONDS.sleep(2);
+        driver.findElement(By.id("com.android.settings:id/password_entry")).sendKeys("1234\n");
+        TimeUnit.SECONDS.sleep(2);
+        driver.findElement(By.id("com.android.settings:id/password_entry")).sendKeys("1234\n");
+        TimeUnit.SECONDS.sleep(2);
+        driver.findElement(By.id("com.android.settings:id/next_button")).click();
+        TimeUnit.SECONDS.sleep(2);
+        driver.quit();
+    }
+
+    /**
+     * add a new finger print to security.
+     */
+    @Test public void pressKeyCodeTest() throws Exception {
+        try {
+            final AndroidDriver driver = getAndroidDriver(".fingerprint.FingerprintSettings");
+            TimeUnit.SECONDS.sleep(2);
+            driver.findElement(By.id("com.android.settings:id/password_entry")).sendKeys("1234\n");
+            TimeUnit.SECONDS.sleep(2);
+            driver.findElement(
+                By.xpath("//android.widget.ListView[1]/android.widget.LinearLayout[1]")).click();
+            TimeUnit.SECONDS.sleep(2);
+            driver.fingerPrint(2);
+            driver.findElement(By.id("com.android.settings:id/next_button")).click();
+            TimeUnit.SECONDS.sleep(2);
+            driver.quit();
+        } catch (Exception ignored) {
+            Assert.fail();
+        }
+    }
+
+    /**
+     * disabling pin lock mode.
+     */
+    @After public void after() throws Exception {
+        final AndroidDriver driver = getAndroidDriver("ChooseLockGeneric");
+        TimeUnit.SECONDS.sleep(2);
+        driver.findElement(By.id("com.android.settings:id/password_entry")).sendKeys("1234\n");
+        TimeUnit.SECONDS.sleep(2);
+        driver.findElement(By.xpath("//android.widget.ListView[1]/android.widget.LinearLayout[1]"))
+            .click();
+        TimeUnit.SECONDS.sleep(2);
+        driver.findElement(By.id("android:id/button1")).click();
+        TimeUnit.SECONDS.sleep(2);
+        driver.quit();
+    }
+
+    private AndroidDriver getAndroidDriver(String activity) throws Exception {
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+        capabilities.setCapability(MobileCapabilityType.DEVICE_NAME, "Android Emulator");
+        capabilities.setCapability("appPackage", "com.android.settings");
+        capabilities.setCapability("appActivity", activity);
```

Comment:
@alizelzele Maybe ApiDemos has convenient functionality. Have you tried to play with it?

Reply:
@TikhomirovSergey searched it. did not find anything fingerprint related to use.